modules/custom/az_core/src/Form/QuickstartCoreSettingsForm.php

```diff
@@ -149,6 +149,19 @@ public function buildForm(array $form, FormStateInterface $form_state) {
       '#field_prefix' => $this->requestContext->getCompleteBaseUrl(),
     ];
 
+    $form['marketing_page'] = [
+      '#type' => 'details',
+      '#title' => t('Marketing Campaign Pages'),
+      '#open' => TRUE,
```

Comment:
@joeparsons Thanks for the comment! I agree: the config setting should be owned by the Flexible Page module. I have made this change. I thought it might make sense to have the new Flexible Page settings form require the "administer content types" permission (which is Administrator only), but I'm happy to change it if another permission would be better!